website_and_docs/content/documentation/test_practices/encouraged/fresh_browser_per_test.en.md

```diff
@@ -14,6 +14,59 @@ If spinning up a new virtual machine is not practical,
 at least start a new WebDriver for each test.
 Most browser drivers like GeckoDriver and ChromeDriver will start with a clean
 known state with a new user profile, by default.
+
+A new browser per test can be achieved by using a test framework's "before each test" hook or fixture. This also implies using the "after each test" hook to close the browser.
+
+```java
+// Using a class variable
+public abstract class BaseTest {
+	protected WebDriver driver;
+    ...
+
+    // Before each test hook
+    public void setupTest() {
+        driver = new FirefoxDriver();
+        ...
+    }
+
+    // After each test hook
+    public void teardownTest() {
+        ...
+        driver.quit();
+    }
+}
+```
+
+```python
+# Using python fixtures
+@pytest.fixture(autouse=True, scope='function')
+def driver(self, request, page: Page):
+    # Create the driver
+    driver = webdriver.Firefox()
+    
+    # Return control to the test 
+    yield self.driver
+
+    # Test ends driver quits
+    driver.quit()
+```
```

Comment:
There is already a code sample for this: https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/python/tests/conftest.py#L15-L21